packages/yarnpkg-core/sources/execUtils.ts

```diff
@@ -51,7 +51,10 @@ export async function pipevp(fileName: string, args: Array<string>, {cwd, env =
 
   const child = crossSpawn(fileName, args, {
     cwd: npath.fromPortablePath(cwd),
-    env,
+    env: {
+      ...env,
+      PWD: npath.fromPortablePath(cwd),
```

Comment:
In PowerShell / CMD there is no `PWD` environment variable. Adding it unconditionally might be dangerous if tools check the existence of that variable. I honestly don't know if any tools do that though&hellip;